scripts/snakenet-migration/index.js

```diff
@@ -205,6 +214,8 @@ const all_modules = [
     ["GenesisHistory.PreviousChain", "0x1754677a24055221d22db56f83f5e21390895d6c6b21a85c004b8942c3bc35ae"],
 ]
 
+const excludeFromTripling = ["7NPoMQbiA6trJKkjB35uk96MeJD4PGWkLQLH7k7hXEkZpiba"];
```

Comment:
what is this account?

GalactiCouncil `7HqdGVRB4MXz1osLR77mfWoo536cWasTYsuAbVuicHdiKQXf` and treasury should be excluded, we can add Alice, but it should be noted that is just for fork

Reply:
No particular meaning. it was only for testing puporses, same as Alice address used. I will clean it up.